loadtest/apigee/go.mod

```diff
@@ -2,9 +2,13 @@ module github.com/apigee/apigee-remote-service-envoy/loadtest/apigee
 
 go 1.16
 
+// Viper pulls in github.com/hashicorp/hcl which has a MPL license.
+// We don't need or use this library, so replace it with a local shim.
+replace github.com/hashicorp/hcl => ../../hcl_shim
```

Comment:
Was this necessary to avoid the dependency?

Reply:
Yeah this made the deps `github.com/hashicorp/hcl` disppear from `go.sum`.